wiki_docs/Create How_GTM_works

```diff
@@ -0,0 +1,47 @@
+User can add or manage the AdWords, Google analytics, Flood-light and non-Google tags using the GTM.
+
+How it works
+  Followings are main components of the GTM:
+
+Account
+Each account can contains multiple containers
+  "One account per company/organization"
+
+Container
+A container includes the tracking tags (e.g. Google Analytics, AdWords Conversion Tracking tags) that you want to use on the website.
+  "One container for one website"
```

Comment:
You can say: "One website should have one container. If you have a dev or staging environment for your website, you should have a separate container for those URLs."